users/test/test_views.py

```diff
@@ -5,16 +5,105 @@
 from faker import Faker
 from rest_framework import status
 from rest_framework.reverse import reverse
-from rest_framework.test import APIRequestFactory, APITestCase
+from rest_framework.test import (
+    APIRequestFactory,
+    APITestCase,
+    force_authenticate,
+)
 
-from users.views import PasswordResetAPIView, PasswordResetEmailView
+from users.views import (
+    PasswordResetAPIView,
+    PasswordResetEmailView,
+    UserFollowView,
+)
 
 fake = Faker()
 User = get_user_model()
+follow_view = UserFollowView.as_view()
 password_reset_email_view = PasswordResetEmailView.as_view()
 password_reset_api_view = PasswordResetAPIView.as_view()
 
 
+class TestFollowingView(APITestCase):
+    def setUp(self) -> None:
+        self.user_one = User.objects.create(
+            username=fake.name(), email=fake.email(), password=fake.password()
+        )
+        self.user_two = User.objects.create(
+            username=fake.name(), email=fake.email(), password=fake.password()
+        )
+        self.factory = APIRequestFactory()
+
+    def test_unauthorized_user_follow(self) -> None:
+        url = reverse("user-follow", kwargs={"pk": self.user_one.id})
```

Comment:
Use uuid instead of the sequential id

Reply:
Rueben is implementing this. We will still use the variable 'id' and not 'uuid'